tests/connection/test_connection_service.py

```diff
@@ -21,9 +21,12 @@
 from ossdbtoolsservice.utils import constants
 from ossdbtoolsservice.utils.cancellation import CancellationToken
 from ossdbtoolsservice.workspace import WorkspaceService
+from ossdbtoolsservice.driver.types.psycopg_driver import PostgreSQLConnection
 from tests.integration import get_connection_details, integration_test
 import tests.utils as utils
-from tests.utils import MockConnection, MockCursor, MockRequestContext
+from tests.utils import MockCursor, MockRequestContext
+from tests.utils import MockConnection as MockPsycopgConnection
```

Comment:
Given that we are doing "from tests.utils import MockConnection as MockPsycopgConnection" at all the places, we should just rename the class. It will make it easier to read as well